preview-site.sh

```diff
@@ -9,30 +9,20 @@ mkdir $PREVIEW_DIR
 cp -rf vitess.io/* $PREVIEW_DIR
 
 mkdir $PREVIEW_DIR/_includes/doc
+mkdir -p $PREVIEW_DIR/posts/
 mkdir -p $PREVIEW_DIR/_posts/doc
 
-# create ref files for each doc
```

Comment:
Removing this breaks our `vitess.io/doc/` folder which we've been relying on to link to docs. For example, we link to `vitess.io/doc/VTGateV3Features` in some places. If we want to make those autogenerated docs not be "posts" anymore, we should convert them to autogenerated "pages" that live in `vitess.io/doc/`. I think that might mean we should check-in those generated files in the vitess/vitess.io/doc/ folder of our repo, like we do for other pages. Unfortunately with the way our scripts are set up, that would require a separate script that you run to update /doc/ and commit it, then run the publish. That's because publish-site.sh requires all changes to checked-in files to be committed so it can switch to the gh-pages branch. This is getting complicated...